scripts/docker/tox-node.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+cd tox-node
+cargo build --release
```

Comment:
You should build the binary inside **build** stage. And you build it inside **run** stage

Reply:
Yes. I am working on multi-stage build now.